schema.graphql

```diff
@@ -52,10 +52,8 @@ type NftSettings @entity {
   requiredBasketManager: String
 
   allowedAssetTokens: [AllowedAssetToken!]! @derivedFrom(field: "nftSettings")
-  assetDepositMin: BigInt
-  assetDepositMax: BigInt
 
-  maxNfts: [NftLimits!]! @derivedFrom(field: "nftSettings")
```

Comment:
This is named maxNfts in the contract - please keep naming consistent so we know what var it relates to in the contract